test/strings/strcat.c

```diff
@@ -0,0 +1,19 @@
+
+#include <string.h>
+#include "smack.h"
+
+// @flag --unroll=5
```

Comment:
This is the default unroll for this folder as you specified in `config.yml`. So remove this line from regressions that require unroll of 5. It is redundant.